.travis.yml

```diff
@@ -59,6 +59,7 @@ jobs:
         - travis_wait make test-source-e2e
         - travis_wait make test-json-format-output
         - travis_wait make test-odo-config
+        - travis_wait make test-odo-url-int
```

Comment:
s/travis_wait make test-odo-url-int/travis_wait make test-cmd-url
Make more sense i guess

Reply:
I guess i can do that, but how would that make more sense than what is already there. Also, I am in favor of keeping test type in the name, making it explicit that integration test will be run and nothing else

Reply:
Right, that is way i suggested the name ```test-cmd-url```. The name it self says that you can not run e2e test over a command (e2e test runs on user specified scenario where you can run  multiple command/sub-command to achieve it). Integration test always runs on command/sub-command. 

So no need to over qualify the name ;)